core/domain/stats_services.py

```diff
@@ -122,6 +169,14 @@ def record_answers(
         submitted_answer_list):
     """Optimally record a group of answers using an already loaded exploration..
     The submitted_answer_list is a list of SubmittedAnswer domain objects.
+
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: int. The version of the exploration.
```

Comment:
For the function record_answer() I checked the input post request. The version is represented as int. record_answer() internally depends record_answers(). And get_state_answers() and get_sample_answers() don't seem to have any callsites in Oppia source. I did use grep for verifying it. So in this case how to verify the exploration_version's type for functions get_state_answers() and get_sample_answers()?